core/src/main/scala/com/gu/tableversions/core/model.scala

```diff
@@ -37,6 +42,32 @@ object Partition {
 
   case class ColumnValue(column: PartitionColumn, value: String)
 
+  private val ColumnValueRegex =
+    """(?x)
+      |([a-z][a-z0-9_]*)  # column name
```

Comment:
cant remember whether java/scala supports non-greedy, but if it does,  but maybe change `*` to `*?`, to avoid `a=1/b=2` being also a complete match as well

Reply:
Interesting. This regex is only used to each part of a string that has already been split by `/` so I think we're OK here, might not be worth changing?

Reply:
Probably good enough then since already split around `/` indeed